docs/src/components/HeaderNavBar.tsx

```diff
@@ -0,0 +1,253 @@
+                backgroundColor: 'initial',
+                outline: 'initial',
+              },
+            },
+          }}
+          {...props}
+        >
+          <Box px={2}>{icon}</Box>
+          <Box>
+            <Typography color="grey.900" variant="body2" fontWeight={700}>
+              {name}
+            </Typography>
+            <Typography color="text.secondary" variant="body2">
+              {description}
+            </Typography>
+          </Box>
+        </Box>
+      </NextLink>
+    );
+  },
+);
+
+export default function HeaderNavBar() {
+  const [subMenuOpen, setSubMenuOpen] = React.useState(false);
+  const [subMenuIndex, setSubMenuIndex] = React.useState<number | null>(null);
+  const productsMenuRef = React.useRef<HTMLDivElement | null>(null);
+  React.useEffect(() => {
+    if (typeof subMenuIndex === 'number') {
+      document.getElementById(PRODUCT_IDS[subMenuIndex])?.focus();
+    }
+  }, [subMenuIndex]);
+  function handleKeyDown(event: React.KeyboardEvent) {
+    if (event.target !== productsMenuRef.current && event.key === 'Tab') {
+      event.preventDefault();
+      return;
+    }
+    if (event.key === 'ArrowDown') {
+      if (event.target === productsMenuRef.current) {
+        setSubMenuOpen(true);
+      }
+      setSubMenuIndex((prevValue) => {
+        if (prevValue === null) return 0;
+        if (prevValue === PRODUCT_IDS.length - 1) {
+          return 0;
+        }
+        return prevValue + 1;
+      });
+    }
+    if (event.key === 'ArrowUp') {
+      setSubMenuIndex((prevValue) => {
+        if (prevValue === null) return 0;
+        if (prevValue === 0) {
+          return PRODUCT_IDS.length - 1;
+        }
+        return prevValue - 1;
+      });
+    }
+    if (event.key === 'Escape') {
+      setSubMenuOpen(false);
+      setSubMenuIndex(null);
+    }
+  }
+  return (
+    <Navigation>
+      <ul role="menubar">
+        <li
+          role="none"
+          onMouseOver={() => setSubMenuOpen(true)}
+          onFocus={() => setSubMenuOpen(true)}
+          onMouseOut={() => setSubMenuOpen(false)}
+          onBlur={() => setSubMenuOpen(false)}
+        >
+          <div
+            role="menuitem"
+            tabIndex={0}
+            id="products-menu"
+            ref={productsMenuRef}
+            aria-haspopup
+            aria-expanded={subMenuOpen ? 'true' : 'false'}
+            onKeyDown={handleKeyDown}
+          >
+            Products
+          </div>
+          <Popper
+            open={subMenuOpen}
+            anchorEl={productsMenuRef.current}
+            transition
+            placement="bottom-start"
+          >
+            {({ TransitionProps }) => (
+              <Fade {...TransitionProps} timeout={350}>
+                <Paper
+                  variant="outlined"
+                  sx={{
+                    minWidth: 498,
+                    overflow: 'hidden',
+                    boxShadow: '0px 4px 20px rgba(170, 180, 190, 0.3)',
```

Comment:
Is this a design token or a one-off style? I mean, we didn't configure the theme's elevation.

Reply:
yes, we don't do that yet. will talk with @danilo-leal about the shadows but in this PR will keep it like this.